src/com/pentalog/pentastagiu/homework2/PR3/Board.java

```diff
@@ -0,0 +1,208 @@
+package com.pentalog.pentastagiu.homework2.PR3;
+
+
+import java.util.Scanner;
+
+public class Board {
```

Comment:
ar fi 2 refcatorizari pe care ti le-as sugera in clasa asta:
1. sa separi logica de citire de la consola a datelor de cea din Board; e principiul "separation of concern"; astfel daca apoi doresti sa inlocuiesti citirea de la consola a datelor cu obtinerea lor din partea de front end a unei aplicatii web, o sa iti fie foarte usor; cu implementarea ta ar trebui sa rescrii clasa asta astfel incat sa fie adaptata noului loc de unde se obtin datele
2. se incerci sa refactorizezi metodele astfel incat sa nu contina f multe linii de cod; daca o metoda contine multe linii de cod(aici e discutabil ce inseamna asta, unii zic ca daca sunt mai mult de 5 linii ar trebui deja refactorizat) e greu de urmarit logica de acolo